test/integration/patients/worklist/schedule.js

```diff
@@ -680,12 +692,24 @@ context('schedule page', function() {
       .get('[data-filter-button')
       .should('contain', 'Medicare');
 
+    cy
+      .get('@filtersSidebar')
+      .find('[data-filter-button]')
+      .eq(2)
+      .get('.sidebar__label')
+      .should('contain', 'Team');
+
     cy
       .get('@filtersSidebar')
       .find('[data-filter-button]')
       .first()
       .click();
 
+    cy
+      .get('.picklist')
+      .find('.js-input')
+      .should('have.attr', 'placeholder', 'Group...');
```

Comment:
Simply confirms that `isSelected: true` input exists in the droplist and that the correct placeholder is used (name of the custom filter).